src/main/resources/log4j.properties

```diff
@@ -1,5 +0,0 @@
-log4j.rootLogger=WARN, console
```

Comment:
What happened to this logging config then if the other one is a test config?

Reply:
As I said: This was never used anywhere because this is a library and not a docker artifact